src/js/vault-client/vault-api.js

```diff
@@ -58,12 +59,18 @@ const fetchVaultToken = async () => {
 
 export const initVaultPwd = (idep) => {
 	axiosVault(`${VAULT_BASE_URI}/v1/${VAULT_KV_ENGINE}/data/${idep}/.onyxia/profile`)
-		.then(({ data: { data : {data}} }) => store.dispatch(newVaultData(data)))
+		.then(({ data : {data} }) => 
+			pwdMustBeRenewed(data.metadata.created_time) ? resetVaultPwd(idep) : store.dispatch(newVaultData(data.data))
+			)
 		.catch(() => {
 			resetVaultPwd(idep);
 		});
 };
 
+const pwdMustBeRenewed = (created_time) => {
```

Comment:
I dislike the `snake_case` convention. But that may be personal preference. Anyway, onyxia is mostly using `lowerCamelCase` at this point, I suggest we try to follow this convention as much as possible.